nbbuild/antsrc/org/netbeans/nbbuild/extlibs/licenseinfo/CommentType.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.nbbuild.extlibs.licenseinfo;
+
+public enum CommentType {
```

Comment:
Not necessarily in this patch, but we may need to add ability to add URLs here, pointing to either ASF, incubator or our FAQs, so that we can add more detailed explanation. Out of the 4 below, the only one that may be understandable without specific knowledge is COMMENT_UNSUPPORTED. (I think I understand what are those used for, but not sure if reviewers will without explanation.)